### PR TITLE
chore: Remove old prompts work

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

We are consistently calling `my_prompts` despite no longer building on this concept. Pretty clear it is dead in the water and worth removing.

## Changes

* Removes frontend and backend work for this

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: posthog/migrations/0389_remove_prompts.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Generated by Django 4.1.13 on 2024-02-09 11:57
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```
Comment (pauldambra) [inline posthog/migrations/0389_remove_prompts.py:6]: maybe we just mark the model as deprecated to avoid messing about with the migration?

**Review by pauldambra — APPROVED** [review]
You can probably add an override for the drop table warning but might be easier to mark the model as deprecated instead

File: posthog/migrations/0389_remove_prompts.py (hunk 1 above)
Comment (benjackwhite) [reply posthog/migrations/0389_remove_prompts.py:6]: eugh... I hate leaving dead code lying around for no reason though...

Comment (pauldambra) [reply posthog/migrations/0389_remove_prompts.py]: `test_migrations_are_safe.py` is your friend then....

IIRC I did write tests the last time I added an override to it

You're welcome to ahab this whale 🙈

**Review by raquelmsmith — APPROVED** [review]
Thanks, Ben! Slight ☹️ from me since I like the idea of these, but I feel like it should be built as a proper product if we're going to have it.

**Pull request merged**